MultiQC/1.5/MultiQC.nf

```diff
@@ -17,6 +18,6 @@ process MultiQC {
     script:
     """
 
-    multiqc ${params.multiqc.toolOptions} $params.out_dir 
+    multiqc ${params.optional} $out_dir
```

Comment:
Why do you need an out_dir? 

https://github.com/UMCUGenetics/NextflowModules/blob/be1ebdf70196f63ccadc1b92855639451561e678/MultiQC/1.8/MultiQC.nf